### PR TITLE
Halley's greeting

### PR DESCRIPTION
**Pull request by halleypaulson**
Adds a method for Halley's greeting

### CODE REVIEW

**Review by lam00071 — DISMISSED** [review]
:D Great!

**Review by JadynSondrol — DISMISSED** [review]
Looks good!

**Review by NicMcPhee — DISMISSED** [review]

**Review by JadynSondrol — DISMISSED** [review]

**Review by NicMcPhee — DISMISSED** [review]

**Review by JadynSondrol — DISMISSED** [review]

**Review by NicMcPhee — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -23,6 +23,7 @@ public String generateOutput() {
     builder.append(aliceSaysHello());
     builder.append(cassieSaysHello());
     builder.append(danteSaysHello());
+    builder.append(halleySaysHi());
```
Comment (lam00071) [inline src/main/java/hellos/Hellos.java:26]: I dont know what else to cmt so let's get this done! :D

**Review by NicMcPhee — APPROVED** [review]

**Review by lam00071 — APPROVED** [review]
lets gooooo

**Pull request merged**